apps/@sparrow-desktop/src/models/tab.model.ts

```diff
@@ -361,7 +356,95 @@ export const tabSchemaLiteral = {
           },
           required: ["variable", "type", "state"],
         },
-        webcocket: {
+        websocket: {
+          type: "object",
+          properties: {
+            url: {
+              type: "string",
+            },
+            headers: {
+              type: "array",
+              items: {
+                type: "object",
+                properties: {
+                  key: {
+                    type: "string",
+                  },
+                  value: {
+                    type: "string",
+                  },
+                  checked: {
+                    type: "boolean",
+                  },
+                },
+              },
+            },
+            queryParams: {
+              type: "array",
+              items: {
+                type: "object",
+                properties: {
+                  key: {
+                    type: "string",
+                  },
+                  value: {
+                    type: "string",
+                  },
+                  checked: {
+                    type: "boolean",
+                  },
+                },
+              },
+            },
+            autoGeneratedHeaders: {
+              type: "array",
+              items: {
+                type: "object",
+                properties: {
+                  key: {
+                    type: "string",
+                  },
+                  value: {
+                    type: "string",
+                  },
+                  checked: {
+                    type: "boolean",
+                  },
+                },
+              },
+            },
+            message: {
+              type: "string",
+            },
+            state: {
+              type: "object",
+              properties: {
+                socketNavigation: {
+                  type: "string",
+                },
+                socketMessageLanguage: {
+                  type: "string",
+                },
+                socketLeftSplitterWidthPercentage: {
+                  type: "string",
+                },
+                socketRightSplitterWidthPercentage: {
+                  type: "string",
+                },
+                isSaveSocketInProgress: {
+                  type: "string",
+                },
+                isParameterBulkEditActive: {
+                  type: "string",
+                },
+                isHeaderBulkEditActive: {
+                  type: "string",
+                },
+              },
+            },
+          },
+        },
+        socketio: {
```

Comment:
here also we don't have events in socketio

Reply:
resolved